dev/serve.vue

```diff
@@ -25,13 +25,15 @@ export default defineComponent({
       rel="stylesheet"
       href="https://unpkg.com/@creativecommons/vocabulary@2020.11.3/css/vocabulary.css"
     />
-    <cc-explore
-      donation-url="https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&c_src2=top-of-page-banner"
-    />
-    <cc-global-header
-      base-url="http://127.0.0.1:8000"
-      donation-url="https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&amp;c_src2=NavBar"
-    />
+    <div class="container">
```

Comment:
Let's include a note in our developer documentation about wrapping the CC global components in a `container` div.